src/meta_schedule/task_scheduler/gradient_based.cc

```diff
@@ -178,6 +201,7 @@ class GradientBasedNode final : public TaskSchedulerNode {
     }
     record.best_time_cost_history.push_back(best_time_cost);
     record.trials += results.size();
+    TVM_PY_LOG(CLEAR, this->logging_func);
```

Comment:
we might want to decouple the `TVM_PY_LOG` macro with screen clearing, in case one might misuse for example:

```suggestion
    TVM_PY_LOG(CLEAR, this->logging_func) << "I'm expecting to print anything";
```

Reply:
How about:

```python
TVM_PY_LOG_CLEAR_SCREEN(this->logging_func);
```